66_HLSLBxDFTests/app_resources/tests.hlsl

```diff
@@ -91,18 +154,21 @@ struct SBxDFTestResources
         retval.rng = nbl::hlsl::Xoroshiro64Star::construct(seed);
         retval.u = float32_t3(rngUniformDist<float32_t2>(retval.rng), 0.0);
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/165#discussion_r1906863399